cmd/notify-mailer/main_test.go

```diff
@@ -26,26 +26,36 @@ func TestCheckpointIntervalOK(t *testing.T) {
 		{interval{end: 10}},
 		{interval{start: 10, end: 15}},
 	}
-	for _, testcase := range okCases {
-		err := testcase.testInterval.ok()
-		test.AssertNotError(t, err, "valid interval produced ok() error")
+	for _, tc := range okCases {
+		t.Run(fmt.Sprintf("interval %v", tc.testInterval), func(t *testing.T) {
+			err := tc.testInterval.ok()
+			// NOTE: Do we need subtests in cases where there are only assertions?
```

Comment:
Rather than modifying AssertNotError, I would change these to the more traditional Go style `if err != nil { t.Errorf`. I think we tend to overuse AssertNotError in our unittests!